clcache.py

```diff
@@ -313,26 +313,17 @@ def section(self, key):
     def sections(self):
         return (CompilerArtifactsSection(path) for path in childDirectories(self._compilerArtifactsRootDir))
 
-    def removeObjects(self, stats, removedObjects):
-        for o in removedObjects:
-            dirPath = self.section(o).cacheEntryDir(o)
-            if not os.path.exists(dirPath):
-                continue  # May be if object already evicted.
-            objectPath = os.path.join(dirPath, "object")
-            if os.path.exists(objectPath):
-                # May be absent if this if cached compiler
-                # output (for preprocess-only).
-                fileStat = os.stat(objectPath)
-                stats.unregisterCacheEntry(fileStat.st_size)
```

Comment:
Ah, so

1. The method could be "private" now (i.e. get a leading underscore) and
2. It's worth researching why `removedItems` in `postprocessHeaderChangeMiss` is unused. It looks like this happened in 39765c24 which apparently also renderes the `MAX_MANIFEST_HASHES` constant unused. Does this mean that the size of manifest is no longer bounded? I didn't notice that... but I guess it's actually a good thing (otherwise two invocations might be considered equal if the first 100 includes are the same) and shouldn't hurt.